src/modules/vault/hooks/details/useVaultDetails.ts

```diff
@@ -24,7 +27,9 @@ const useVaultDetails = () => {
     APIToken,
     commingSoonFeatures: { commingSoonDialog, selectedFeature },
   } = useCLI({
-    vaultId: vaultRequest.data.id ?? '',
+    vault: data?.predicates?.data.find(
+      (predicate) => predicate.id === vaultRequest?.data?.id,
+    ),
```

Comment:
this should fetch the Vault information by ID from the API (https://github.com/infinitybase/bako-safe-api/pull/213). 
this implementation may cause other bugs when the vault is not listed